src/Components/OATPropertyEditor/Internal/PropertiesModelSummary.tsx

```diff
@@ -351,8 +376,30 @@ export const PropertiesModelSummary: React.FC<IPropertiesModelSummaryProps> = (
                             >
                                 {t('OATPropertyEditor.version')}
                             </Text>
-                            <SpinButton
+                            <TextField
+                                aria-labelledby={'oat-model-path'}
+                                onBlur={() => commitModelIdChange(itemId)}
+                                onChange={onChangeVersion}
+                                styles={
+                                    classNames.subComponentStyles.stringField
+                                }
+                                value={modelVersion}
+                            />
+                            {/* <SpinButton
```

Comment:
remove if not used. I see that the other PR is connected to this one, so maybe intentional...